### ISSUE TITLE
No audio cue played when stopping dictation

### ISSUE THREAD

**Issue by deepak1556** [microsoft/vscode#208559]
Testing #207642

I was able to confirm the audio cue on dictation start in editor, terminal and chat but when stopping the dictation no feedback is heard.

**Comment by meganrogge** [comment]
Note that sometimes `accessibility.signals.voiceRecordingStopped` will not play because of a content media policy which blocks media from playing if it wasn't initiated via user gesture (IE within a certain amount of time from the gesture). We cannot fix this, see discussion here https://github.com/microsoft/vscode/issues/178642#issuecomment-1840446946.

**Comment by eleanorjboyd** [comment]
also not hearing the stop dictation but not entirely sure if it is possible in the context I tried it in due to the constraints.

**Comment by meganrogge** [comment]
If you start/stop it fast enough, you'll hear it. We cannot fix this, see discussion here https://github.com/microsoft/vscode/issues/178642#issuecomment-1840446946.

**Comment by deepak1556** [comment]
@meganrogge I am confused why autoplay is involved here, when running the commands to stop dictation or clicking the mic icon in chat to stop dictation can we not use that as top level user gesture and play the audio element on demand ?

**Comment by meganrogge** [comment]
@deepak1556 that is a good point. I had implemented this here to avoid calling it in many different places, but maybe I need to do that 🤔 cc @bpasero 

https://github.com/microsoft/vscode/blob/9ed28b5d674d01f475c98dd6a77139aaa838f291/src/vs/workbench/contrib/speech/browser/speechService.ts#L157

**Comment by meganrogge** [comment]
@bpasero noticed that this is off by default. 🤦‍♀ @deepak1556 can you repro if you set it to `on`?
